ES/ESmy015.xml

```diff
@@ -0,0 +1,513 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="ESmy015" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Story of the Trinity</title>
+            <editor role="cataloguer" key="VP"/>
+            <editor key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+            <date>2016-06-07T17:45:18.367+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0039MY"/>
+                  <collection>Ethio-SPaRe</collection>
+                    <idno>MY-015</idno>
+
+               </msIdentifier>
+               <msContents>
+                <summary/>
+                <msItem xml:id="ms_i1">
+                   <locus from="5ra" to="43ra"></locus>
+                   <title ref="LIT2296Sayfas">ዜና፡ ነገሮሙ፡ ለስላሤ፡</title>
+                   <note>The work is known under the title Sayfa śəllāse
+                      “Sword of the Trinity”. The content of the manuscript seems to be similar to Mss <ref type="mss" target="ANLcr53">StrANL Conti Rossini 53</ref> (<bibl>
```

Comment:
are the IDs ANLcr53 etc. present in the BM database, at least as stubs? the links seem to go nowhere...